www/faq.md

```diff
@@ -0,0 +1,53 @@
+# FAQs
+
+If you don't find the answer to your question here, please contact Natalie Kucher at nkucher3@jhu.edu!
```

Comment:
Should we make this look like nkucher3 at jhu.edu ? So she doesn't get spammed?

Reply:
Yes, good idea!